src/main/scala/uk/gov/hmrc/ct/accounts/frs10x/abridged/relatedPartyTransactions/RelatedPartyTransactions.scala

```diff
@@ -59,9 +60,15 @@ case class RelatedPartyTransactions(transactions: List[RelatedPartyTransaction]
     }
   }
 
+  def validateFields(boxRetriever: AbridgedAccountsBoxRetriever)() = {
+    ac7806.validate(boxRetriever).map {
+      error => error.copy(boxId = Some("RelatedPartyTransactions"))
+    }
+  }
+
   private def contextualiseErrorKey(errorKey: String, context: String): String = {
     val splitKey = errorKey.split('.')
-    (splitKey.take(1) ++ Array("transactions") ++ Array(context) ++ splitKey.drop(1)).mkString(".")
+    (splitKey.take(1) ++ Array("compoundList", "transactions") ++ Array(context) ++ splitKey.drop(1)).mkString(".")
```

Comment:
Could we get a comment or break some function out this to show what its doing, maybe an example of the transformation from one key to the other in a comment would do